react/features/toolbox/components/web/Toolbox.tsx

```diff
@@ -366,7 +366,6 @@ const styles = () => {
             margin: 0,
             marginBottom: '8px',
             maxHeight: 'calc(100vh - 100px)',
-            width: '240px'
```

Comment:
Update to `min-width` instead of removing it, please

Reply:
Changed it to min-width.